### ISSUE TITLE
Does anyone have an example of running the the example notebook with "conv" model type?

### ISSUE THREAD

**Issue by rbunn80110** [amiratag/DataShapley#7]
I'm trying to work out the changes necessary to run the example notebook with the model type of conv.  If anyone has any examples of this already created it would be super helpful.

Thanks,

Bob

**Comment by shachi01** [comment]
@rbunn80110, I am currently working on it. But I guess model for 'conv' hasn't been built fully. Could see comment like "FIXIT" in the code.
Have you implemented it?

Thanks,
Shachi

**Comment by rbunn80110** [comment]
No, I haven't had time to work on this.  If I ever do I'll update this thread.

**Comment by ffeiland** [comment]
Did you already solve this problem?

Thanks,
Fabian

**Comment by rbunn80110** [comment]
No, I stopped working on this because it simply felt like it’s currently an incomplete project.  If their technique could be applied beyond toy problems then I suspect they would have wrote the code to do that already.  I suspect it’s simply not as good as they say for real world problems.
